source/Thumb.js

```diff
@@ -83,11 +98,11 @@
 				}
 				var sbo = inStrategy[this.positionMethod]() - over;
 				// calc size & position
-				var bdc = bd - this.cornerSize;
+				var bdc = bd - moon.riScale(this.cornerSize);
```

Comment:
Good call - these are not changed internally and are private, so they shouldn't be changed by the user after `create` time - I'll make an update.